src/Umbraco.Web.Common/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -291,6 +296,13 @@ public static IUmbracoBuilder AddWebComponents(this IUmbracoBuilder builder)
             return builder;
         }
 
+        public static IUmbracoBuilder AddUnattedInstallCreateUser(this IUmbracoBuilder builder)
+        {
+            builder.AddNotificationAsyncHandler<UnattendedInstallNotification, CreateUnattendedUserNotificationHandler>();
+            //builder.AddNotificationHandler<UmbracoApplicationStarting, CreateUnattendedUserNotificationHandler>();
```

Comment:
Create the uncommented code

Reply:
I was going to remove it as the original event worked.